docker/Dockerfile.http.dev

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:20.04
+
+# Get arguments from the build command line
+ARG version
+ENV VERSION=$version
+
+# Build up the OS
+RUN export DEBIAN_FRONTEND=noninteractive && \
```

Comment:
There is no point to build custom Ubuntu image. Replace with standard image from Docker Hub, for example `FROM maven:3.8-jdk-11-slim`

Reply:
see above